README.md

```diff
@@ -420,6 +421,93 @@ status:
 - `$.status.conditions[?(@.type == "Failed")].status == "True"` means that a failure condition has been fulfilled and the resource is considered failed.
 - Since `error_msg_path` is specified, kubernetes-deploy will log the contents of `$.status.conditions[?(@.type == "Failed")].message`, which in this case is: `resource is failed`.
 
+### Deploy walkthrough
+
+Let's walk through what happens when you run the `deploy` task with [this directory of templates](https://github.com/Shopify/kubernetes-deploy/tree/master/test/fixtures/hello-cloud). You can see this for yourself by running the following command:
+
+```bash
+krane deploy -f test/fixtures/hello-cloud --render-erb my-namespace my-k8s-cluster
+```
+
+... or the following test:
+
+```bash
+PRINT_LOGS=1 bundle exec ruby -I test test/integration/kubernetes_deploy_test.rb --name test_full_hello_cloud_set_deploy_succeeds
```

Comment:
Not sure I like the idea of discussing tests in a walk-through.

Reply:
I agree in principle, but I do like the idea of making it super easy to reproduce this. Do you think it would be brittle to changes?

Reply:
Not to changes. I just think its out of place to discuss tests (which also require setting up your local env correctly) in a production walk through. I think its ok to mention the `test_full_hello_cloud_set_deploy_succeeds` does this in CI.